src/test/java/dev/codesupport/testutils/entity/MockIdentifiableEntity.java

```diff
@@ -1,12 +1,16 @@
 package dev.codesupport.testutils.entity;
 
-import dev.codesupport.web.common.service.data.entity.IdentifiableEntity;
+import dev.codesupport.web.common.data.entity.IdentifiableEntity;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
 
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
 public class MockIdentifiableEntity implements IdentifiableEntity<Long> {
 
-    @Override
-    public Long getId() {
-        return 1L;
-    }
+    private Long id;
+    private String propertyA;
```

Comment:
Doesn't seem like a very semantic variable name?

Reply:
It's just a test class